tests/end-to-end/external-solution-integration/chart/external-solution/values.yaml

```diff
@@ -3,7 +3,7 @@ containerRegistry:
 
 image:
   dir: ""
-  version: PR-9354
```

Comment:
PR-9354 was merged in commit 415d68866
would it not make sense to use the image created by this commit?

Reply:
Done! It should make more sense right now ^^ Do you know maybe if we can check this information only from the Container Registry side?